recipes/rasa-sdk/meta.yaml

```diff
@@ -0,0 +1,71 @@
+{% set name = "rasa-sdk" %}
+{% set version = "3.0.0rc3" %}
+{% set build_num = 0 %}
+
+package:
+  name: {{ name|lower }}-meta
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 162e72dc3c141d9f5220f3de921be6c902e03f209980a42cdb2cb1ff21f534d6
+  
+
+build:
+  skip: True  # [osx or win]
+  number: {{ build_num }}
+
+outputs:
```

Comment:
Is it only one package? Then no outputs are needed.

Reply:
It's only one package, but (I think) it also offers extras. So, paving the way, for easily adding the extras should the need arise.